erizo/src/erizo/rtp/RtpHeaders.h

```diff
@@ -420,9 +420,18 @@ namespace erizo{
       }
       inline uint32_t getOctetsSent(){
         return ntohl(report.senderReport.octetssent);
-      }      
+      }
       inline uint64_t getNtpTimestamp(){
-       return (((uint64_t)htonl(report.senderReport.ntptimestamp)) << 32) + htonl(report.senderReport.ntptimestamp >> 32);
+        return (((uint64_t)htonl(report.senderReport.ntptimestamp)) << 32) + htonl(report.senderReport.ntptimestamp >> 32);
+      }
+      inline uint32_t getNtpTimestampMSW() {
+        return htonl((uint32_t)report.senderReport.ntptimestamp);
```

Comment:
Yeah, this is related to the code which was already in rtpheaders, which uses

return (((uint64_t)htonl(report.senderReport.ntptimestamp)) << 32) + htonl(report.senderReport.ntptimestamp >> 32);

I'll investigate. thanks